### PR TITLE
Include Postgres client for debugging purposes

### PR DESCRIPTION
**Pull request by jviotti**
Change-type: patch

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -43,6 +43,12 @@ RUN apt-get update \
 		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+	wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+  apt-get update && \
+  apt-get install -y postgresql-client-11 && \
```
Comment (Page-) [inline Dockerfile]: I think this also needs to go in the no systemd image, cc @roman-mazur?

Comment (roman-mazur) [reply Dockerfile]: Yes, I will appreciate if this can be added to the no systemd variant.
https://github.com/balena-io/open-balena-base/blob/master/Dockerfile.no-systemd#L6

Comment (jviotti) [reply Dockerfile]: Done!

**Comment by jviotti** [comment]
@Page- This is what apt reports:

```
The following NEW packages will be installed:
  netbase pgdg-keyring postgresql-client-11 postgresql-client-common
0 upgraded, 4 newly installed, 0 to remove and 2 not upgraded.
Need to get 1506 kB of archives.
After this operation, 6526 kB of additional disk space will be used.
Get:1 http://apt.postgresql.org/pub/repos/apt buster-pgdg/main amd64 pgdg-keyring all 2018.2 [10.7 kB]
Get:2 http://cdn-fastly.deb.debian.org/debian buster/main amd64 netbase all 5.6 [19.4 kB]
Get:3 http://apt.postgresql.org/pub/repos/apt buster-pgdg/main amd64 postgresql-client-common all 209.pgdg100+1 [86.3 kB]
Get:4 http://apt.postgresql.org/pub/repos/apt buster-pgdg/main amd64 postgresql-client-11 amd64 11.5-3.pgdg100+1 [1390 kB]
```

File: Dockerfile.no-systemd [hunk 2]
```diff
@@ -10,6 +10,7 @@ RUN install_packages \
 		netcat \
 		net-tools \
 		ifupdown \
+    postgresql-client-11 \
```
Comment (roman-mazur) [inline Dockerfile.no-systemd]: @jviotti indentation is wrong here

Comment (jviotti) [reply Dockerfile.no-systemd]: @roman-mazur Fixed! (we should add an `.editorconfig` btw)

File: Dockerfile [hunk 3]
```diff
@@ -43,6 +43,12 @@ RUN apt-get update \
 		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
```
Comment (Page-) [inline Dockerfile]: Why is this needed here but not in the no-systemd image?

Comment (hedss) [inline Dockerfile]: If we're going to add this package, I'd personally rather it was added in the list of all packages in the `RUN` above (and if the sources require adding, doing this as the first line), rather than having this as a separate set of `apt-get` install. It doesn't make much sense for it to be separate.

Comment (hedss) [reply Dockerfile]: Actually, sorry, this might be a silly question. Why's this being added here as opposed to downstream in (I assume) the Jellyfish service that pulls this in as it's parent? Almost all of the children this base image is a parent for won't have use for this, whilst several have use of the other utils we pull in (though looking through the list, I'm starting to wonder if some of these should be extracted, too...).

**Comment by jviotti** [comment]
Closing obsolete PRs...
